src/Package/Impl/DataInspect/Viewers/FunctionViewer.cs

```diff
@@ -33,12 +34,12 @@ internal sealed class FunctionViewer : ViewerBase, IObjectDetailsViewer {
         #region IObjectDetailsViewer
         public ViewerCapabilities Capabilities => ViewerCapabilities.Function;
 
-        public bool CanView(IDebugValueEvaluationResult evaluation) {
-            return evaluation != null && evaluation.Classes.Count == 1 && evaluation.Classes.Any(t => _classes.Contains(t));
+        public bool CanView(IRValueInfo evaluation) {
+            return evaluation != null && evaluation.Classes.Count == 1 && evaluation.Classes[0].EqualsOrdinal("function");
```

Comment:
Bad merge - will fix. Looks like we don't have a test that would catch this, either - should I add one?

---
In reply to: [62136631](https://github.com/Microsoft/RTVS/pull/1731#discussion_r62136631) [](ancestors = 62136631)